env0/data_aws_cost_credentials.go

```diff
@@ -0,0 +1,92 @@
+package env0
+
+import (
+	"context"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataAwsCostCredentials() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataAwsCostCredentialsRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Description:  "the name of the credential",
+				Optional:     true,
+				ExactlyOneOf: []string{"name", "id"},
+			},
+			"id": {
```

Comment:
Why not adding the role arn as well here? one might wanna use the arn of one cost creds in other creds

Reply:
I copied the functionality from the general cred resource, if its needed Ill add it with the changes I am working on